doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -332,6 +335,48 @@ and `Constants`, while implicitly created databases have the `Opaque` setting.
 
    By default, hint databases are undiscriminated.
 
+Hint databases defined in the Coq standard library
+``````````````````````````````````````````````````
+
+Several hint databases are defined in the Coq standard library. The
+actual content of a database is the collection of hints declared
+to belong to this database in each of the various modules currently
+loaded. Especially, requiring new modules may extend the database.
+At Coq startup, only the core database is nonempty and can be used.
+
+:core: This special database is automatically used by ``auto``, except when
+       pseudo-database ``nocore`` is given to ``auto``. The core database
+       contains only basic lemmas about negation, conjunction, and so on.
+       Most of the hints in this database come from the Init and Logic directories.
+
+:arith: This database contains all lemmas about Peano’s arithmetic proved in the
```

Comment:
```suggestion
:arith: all lemmas about Peano’s arithmetic proved in the
```

For consistency with others and to avoid boilerplate "This xxx ..."

Also, how about putting these in alphabetic order?  You could add an "Also see zarith for ..."  in the arith entry.